beard/utils/joblib_helpers.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Joblib library utils.
+
+.. codeauthor:: Mateusz Susik <mateusz.susik@cern.ch>
+
+"""
+
+from joblib import cpu_count
+
+
+def jobs_from_njobs(n_jobs):
+    """Compute number of jobs from Joblib's n_jobs variable.
```

Comment:
This could be written in a more clearer way. We should mention the difference between the given value and the output value, i.e. the fact that the former can be negative while the second is always positive.